ui/views/routing.go

```diff
@@ -0,0 +1,455 @@
+package views
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/jroimartin/gocui"
+	"golang.org/x/text/language"
+	"golang.org/x/text/message"
+
+	"github.com/edouardparis/lntop/config"
+	netmodels "github.com/edouardparis/lntop/network/models"
+	"github.com/edouardparis/lntop/ui/color"
+	"github.com/edouardparis/lntop/ui/models"
+)
+
+const (
+	ROUTING         = "routing"
+	ROUTING_COLUMNS = "routing_columns"
+	ROUTING_FOOTER  = "routing_footer"
+)
+
+var DefaultRoutingColumns = []string{
+	"DIR",
+	"STATUS",
+	"IN CHANNEL",
+	"IN ALIAS",
+	"OUT CHANNEL",
+	"OUT ALIAS",
+	"AMOUNT",
+	"FEE",
+	"LAST UPDATE",
+	"DETAIL",
+}
+
+type Routing struct {
+	cfg *config.View
+
+	columns []routingColumn
+
+	columnsView   *gocui.View
+	view          *gocui.View
+	routingEvents *models.RoutingLog
+
+	ox, oy int
+	cx, cy int
+}
+
+type routingColumn struct {
+	name    string
+	width   int
+	display func(*netmodels.RoutingEvent, ...color.Option) string
+}
+
+func (c Routing) Name() string {
+	return ROUTING
+}
+
+func (c *Routing) Wrap(v *gocui.View) View {
+	c.view = v
+	return c
+}
+
+func (c Routing) currentColumnIndex() int {
+	x := c.ox + c.cx
+	index := 0
+	sum := 0
+	for i := range c.columns {
+		sum += c.columns[i].width + 1
+		if x < sum {
+			return index
+		}
+		index++
+	}
+	return index
+}
+
+func (c Routing) Origin() (int, int) {
+	return c.ox, c.oy
+}
+
+func (c Routing) Cursor() (int, int) {
+	return c.cx, c.cy
+}
+
+func (c *Routing) SetCursor(cx, cy int) error {
+	err := c.columnsView.SetCursor(cx, 0)
+	if err != nil {
+		return err
+	}
+
+	err = c.view.SetCursor(cx, cy)
+	if err != nil {
+		return err
+	}
+
+	c.cx, c.cy = cx, cy
+	return nil
+}
+
+func (c *Routing) SetOrigin(ox, oy int) error {
+	err := c.columnsView.SetOrigin(ox, 0)
+	if err != nil {
+		return err
+	}
+	err = c.view.SetOrigin(ox, oy)
+	if err != nil {
+		return err
+	}
+
+	c.ox, c.oy = ox, oy
+	return nil
+}
+
+func (c *Routing) Speed() (int, int, int, int) {
+	current := c.currentColumnIndex()
```

Comment:
A fix is needed for the cursor. Please, reproduce this change:
https://github.com/edouardparis/lntop/pull/23/files#diff-1a61b9279a10d79e5322cbbc01644da76c7b9e7111de5f718d6fe7a0bc83ec68R140

Reply:
Added fix to bc8aabe95e6982af0feb22b83e9957b36a73a218